spec/support/contracts.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+# # frozen_string_literal: true
+#
+# Contract.validator_strategies.map do |strategy, validator|
```

Comment:
Why is this all commented out?

Reply:
Needs to be deleted. I didn't need test doubles after all.